src/ray/pubsub/publisher.h

```diff
@@ -89,16 +105,19 @@ class Subscriber {
   /// Connect to the subscriber. Currently, it means we cache the long polling request to
   /// memory. Once the bidirectional gRPC streaming is enabled, we should replace it.
   ///
-  /// \param long_polling_reply_callback reply callback to the long polling request.
+  /// \param reply pubsub long polling reply.
+  /// \param send_reply_callback A callback to reply to the long polling subscriber.
   /// \return True if connection is new. False if there were already connections cached.
-  bool ConnectToSubscriber(LongPollConnectCallback long_polling_reply_callback);
+  bool ConnectToSubscriber(rpc::PubsubLongPollingReply *reply,
+                           rpc::SendReplyCallback send_reply_callback);
 
   /// Queue the object id to publish to the subscriber.
```

Comment:
Update this comment.

Reply:
Still needs to be updated.

```suggestion
  /// Queue the pubsub message to publish to the subscriber.
```